app/views/admin/group_users/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="headline">
+  <h1><%= title(t('.title')) %></h1>
+</div>
+
+<div class="col-md-12 margin-bottom-20">
+  <%= link_to t('.all_groups'), admin_groups_path, class: 'btn primary' %>
+</div>
+
+<div class="col-md-12">
+  <%= grid(@grid, class: 'group-users-grid') do |grid|
+    grid.row_attributes do |gu|
+      { id: dom_id(gu), class: gu.fadder ? 'fadder' : '' }
```

Comment:
I think we should remove the spaces inside `{ }`, have caused some errors before.